dump/dump_collect.cpp

```diff
@@ -141,21 +142,38 @@ void collectDumpFromSBE(struct pdbg_target* chip,
                 type, clockState, chipPos, collectFastArray, sbeError.what())
                 .c_str());
 
-        std::string event = "org.open_power.Processor.Error.SbeChipOpFailure";
         auto dumpIsRequired = false;
-
-        if (sbeError.errType() == openpower::phal::exception::SBE_CMD_TIMEOUT)
+        std::string event;
+        if (isOcmb)
```

Comment:
Now SBE_CMD_TIMEOUT will come for ocmb also, so if we make changes here we will have to make all changes to collect SBE dump. which was not done because the SBE dump was not ready when doing it.

I think we can do this as a separate pull request.
for this commit move line 196 above 180

Reply:
done as discussed